src/boot/z_std_dma.c

```diff
@@ -296,9 +296,9 @@ const char* DmaMgr_FindFileName(uintptr_t vrom) {
     }
     //! @bug Since there is no return, in case the file isn't found, the return value will be a pointer to the end
     // of gDmaDataTable
-#ifdef AVOID_UB
+    #ifdef AVOID_UB
     return "";
-#endif
+    #endif
```

Comment:
this one in particular is not ideal id say, but the benefits still outweigh some rarer cases like this i think

Reply:
I agree it kinda looks weird but still more readable than #endif into an unrelated #else